omop_sssom.tsv

```diff
@@ -0,0 +1,49 @@
+# curie_map:
+#   omop: https://data.ohdsi.org/concept/
+#   omop_maps_to_value: https://data.ohdsi.org/maps_to_value/
+#   omoprel: https://data.ohdsi.org/relationship/
+# extension_definitions:
+# - property: omoprel:source_code_description
+#   slot_name: source_code_description
+# - property: omoprel:source_vocabulary_id
+#   slot_name: source_vocabulary_id
+# - property: omoprel:hierarchy
+#   slot_name: hierarchy
+# - property: omoprel:source_concept_class_id
+#   slot_name: source_concept_class_id
+# - property: omoprel:source_vocabulary_id
+#   slot_name: source_vocabulary_id
+# - property: omoprel:source_domain_id
+#   slot_name: source_domain_id
+# - property: omoprel:relationship_group
+#   slot_name: relationship_group
+# - property: omoprel:target_concept_code
+#   slot_name: target_concept_code
+# - property: omoprel:target_concept_value
+#   slot_name: target_concept_value
+# - property: omoprel:target_concept_name
+#   slot_name: target_concept_name
+# - property: omoprel:target_concept_id
+#   slot_name: target_concept_id
+# - property: omoprel:target_concept_class_id
+#   slot_name: target_concept_class_id
+# - property: omoprel:target_domain_id
+#   slot_name: target_domain_id
+# - property: omoprel:target_vocabulary_id
+#   slot_name: target_vocabulary_id
+# - property: omoprel:target_value_concept_code
+#   slot_name: target_value_concept_code
+# - property: omoprel:target_value_concept_value
+#   slot_name: target_value_concept_value
+# - property: omoprel:target_value_concept_name
+#   slot_name: target_value_concept_name
+# - property: omoprel:target_value_concept_class_id
+#   slot_name: target_value_concept_class_id
+# - property: omoprel:target_value_domain_id
+#   slot_name: target_value_domain_id
+# - property: omoprel:target_value_vocabulary_id
+#   slot_name: target_value_vocabulary_id
```

Comment:
## `property` and `slot_name`
There are a lot of new fields here, and I see that they are shorthand for some CURIEs in the OMOP data model. I'm not sure if this `property`/`slot_name` thing is also new with this PR; I don't see it in the docs. But I'm guessing this is not 100% tied to post-coordinated expressions, but some way of extending SSSOM.

Reply:
Its not documented yet, but if you have 10 hours of time to read, here is the thread:

https://github.com/mapping-commons/sssom/issues/328#issuecomment-1879735143

The specific comment I linked is the TLDR for this huuuuuge discussion!

Reply:
(this is not tied to post-coordination, this is a way to ship arbitrary metadata with SSSOM files, i.e. add any column you like to a SSSOM file).